base-libs/gc/autobuild/beyond

```diff
@@ -1,3 +1,4 @@
-sed -i -e 's/GC_MALLOC 3/gc 3/' doc/gc.man
-install -m755 -d "$PKGDIR/usr/share/man/man3"
-install -m644 doc/gc.man "$PKGDIR/usr/share/man/man3/gc.3"
+abinfo "Modifying and installing manual pages ..."
+sed -i -e 's/BDWGC 3/gc 3/' doc/gc.man
```

Comment:
Please use absolute paths - `sed -i -e 's/BDWGC 3/gc 3/' "$SRCDIR"/doc/gc.man`. Ditto for line 4.

Reply:
done